csrankings-q.csv

```diff
@@ -103,6 +103,7 @@ Qinpei Zhao,Tongji University,https://sse.tongji.edu.cn/Data/View/2821,NOSCHOLAR
 Qinping Zhao,Beihang University,http://scse.buaa.edu.cn/info/1084/2894.htm,NOSCHOLARPAGE
 Qinru Qiu,Syracuse University,http://hydrogen.syr.edu/~qqiu,DWv0xG0AAAAJ
 Qiong Luo 0001,HKUST,https://www.cse.ust.hk/~luo,SqAsppUAAAAJ
+Qirong Ho,MBZUAI,https://mbzuai.ac.ae/study/faculty/dr-qirong-ho,tR3AZbwAAAAJ

```

Comment:
Employed as CTO of Petuum; seems incompatible with a full-time academic appointment. Please clarify.